Sources_v3/Controllers/ChannelListController/ChannelListController.swift

```diff
@@ -137,6 +143,29 @@ public class ChannelListControllerGeneric<ExtraData: ExtraDataTypes>: Controller
         where Delegate.ExtraData == ExtraData {
         multicastDelegate.mainDelegate = AnyChannelListControllerDelegate(delegate)
     }
+    
+    // MARK: - Private
+    
+    private func setupEventObservers() {
+        eventObservers = [
+            TypingEventObserver(
+                notificationCenter: client.webSocketClient.eventNotificationCenter,
+                filter: { [unowned self] event in
+                    self.channels.map(\.cid).contains(event.cid)
+                },
+                callback: { [unowned self] event in
+                    self.channelListObserver.reset()
+                    
+                    let index = self.channels.firstIndex(where: { $0.cid == event.cid })!
+                    let change: ListChange = .update(self.channels[index], index: .init(row: index, section: 0))
+
+                    self.delegateCallback {
+                        $0.controller(self, didChangeChannels: [change])
```

Comment:
TBD. Here we manually invoke the list-change for the channel which member started/stopped typing. I find it useful as the SDK users receive all channel updates using `didChangeChannels` method and no extra effort is required from our side to expose typing events